sql/src/main/java/io/crate/operation/join/NestedLoopOperation.java

```diff
@@ -68,6 +68,10 @@
  *             RowReceiver
  * </pre>
  *
+ * In case of Right & Full (Outer) Join once the whole left-right loop is finished a last loop
+ * on the right is executed in order to emit rows from the right that weren't matched with rows
+ * from left.
```

Comment:
So far the NestedLoop also kept the order of the Rows it received. With Right join this is no longer the case - I think that should be mentioned as well